### DISCUSSION TITLE
Tabbar height on WQHD too big

### DISCUSSION THREAD

**Discussion by EbonyBirch** [black7375/Firefox-UI-Fix#419] [Q&A]
Hello,

I've installed the theme on computer with an FHD screen as well as one with WQHD one. On FHD one the tabbar touches the top of the screen so moving cursor to the topmost pixel and clicking switches to that particular tab. On WQHD the tabbar is taller than the tabs themselves so doing the same thing does not change the selected tab. What's worse, it only happens when the window is maximized. Switching to fullscreen with F11 "fixes" the issue (read as: it looks & works as on the FHD screen)

Take a notice where the mouse cursor is and where's the tab outline. Is there a simple way to fix that?

maximized WQHD: https://i.imgur.com/9v7lTlm.png
![image](https://user-images.githubusercontent.com/25581533/175057482-e2c497a0-0a2b-461a-a171-9fabfa9cf3e2.png)

non-maximized WQHD: https://i.imgur.com/DxO3MaH.png
![image](https://user-images.githubusercontent.com/25581533/175057527-fa32cf7c-faa8-4bc5-af11-16276d6f4e3f.png)

FHD (maximized or not): https://i.imgur.com/J9b7X5Q.png
![image](https://user-images.githubusercontent.com/25581533/175057617-0b8fdbad-0751-40c8-a785-b7759f016409.png)

**Comment by black7375** [comment]
I know that the symptoms have appeared in version `101`(above) of Windows, and it is not easy to track even when debugs.
It's a little triky problem.
